features/bootstrap/FeatureContext.php

```diff
@@ -332,11 +332,45 @@ public function getHookDefinitionResources() {
 	 * Replace {VARIABLE_NAME}. Note that variable names can only contain uppercase letters and underscores (no numbers).
 	 */
 	public function replace_variables( $str ) {
-		$ret = preg_replace_callback( '/\{([A-Z_]+)\}/', array( $this, '_replace_var' ), $str );
+		if ( false !== strpos( $str, '{INVOKE_WP_CLI_WITH_PHP_ARGS-' ) ) {
+			$str = $this->_replace_invoke_wp_cli_with_php_args( $str );
```

Comment:
I'm not a fan of leading underscores in method names when we can just mark them as private.
Instead of matching the other functions, maybe consider just changing them to get rid of that underscore in all of them.

Reply:
Yes, looks funny too - will remove underscore from all...